types/underscore/index.d.ts

```diff
@@ -5357,12 +5357,12 @@ declare module _ {
         * Wrapped type `any[]`.
         * @see _.sample
         **/
-        sample<T>(n: number): _Chain<T, T[]>;
+        sample<T>(n: number): _ChainSingle<T>;
```

Comment:
This actually does return an array (unlike  sample<T>())

Reply:
Done. I also added a test --- the <T> here was breaking things.